src/components/Lesson/Lesson.js

```diff
@@ -60,54 +61,69 @@ const Lesson = ({ lesson, slug }) => {
-          Taking the time to go through all of these resources will definitely
-          put you on the road to expert-level knowledge in this subject matter.
-        </Block>
-        {readingLinks && (
+      {readingLinks && (
+        <ContentSection title=" " subTitle="Pre-read Materials">
+          <Block is="p" mb="16px">
+            We've curated these resources to give you greater depth on these
+            subjects. Don't feel like you have to read them all.
+          </Block>
+          <Block is="p" mb="16px">
+            Taking the time to go through all of these resources will definitely
+            put you on the road to expert-level knowledge in this subject
+            matter.
+          </Block>
           <ul className="Lesson-readingList">
             {readingLinks.map(readingLink => (
               <li className="Lesson-readingListItem" key={readingLink.title}>
-                <StyledLink isExternal variation={"tertiary"} path={readingLink.link}>
+                <StyledLink
+                  isExternal
+                  variation={'tertiary'}
+                  path={readingLink.link}
+                >
                   {readingLink.title}
                 </StyledLink>
                 <p>- {readingLink.description}</p>
               </li>
             ))}
           </ul>
-        )}
-      </ContentSection>
-
-      <ContentSection title=" " subTitle="Pre-read Quiz">
-        <Block is="p" mb="16px">
-          This pre-read quiz is designed to challenge your knowledge of the
-          pre-read material. These quizzes are a great way to check your
-          comprehension, and we highly recommend taking them.
-        </Block>
-        <StyledLink
-          path={preReadQuiz !== null ? `${slug}/quiz` : preReadQuizLink}
-          isExternal={preReadQuiz === null}
-          variation={"tertiary"}
-        >
-          Quiz Link
-        </StyledLink>
-      </ContentSection>
+          )}
+        </ContentSection>
+      )}
+      {preReadQuizLink && (
+        <ContentSection title=" " subTitle="Pre-read Quiz">
+          <Block is="p" mb="16px">
+            This pre-read quiz is designed to challenge your knowledge of the
+            pre-read material. These quizzes are a great way to check your
+            comprehension, and we highly recommend taking them.
+          </Block>
+          <StyledLink
+            path={preReadQuiz !== null ? `${slug}/quiz` : preReadQuizLink}
+            isExternal={preReadQuiz === null}
+            variation={'tertiary'}
+          >
+            Quiz Link
+          </StyledLink>
+        </ContentSection>
+      )}
 
-      <ContentSection title=" " subTitle="Exercises">
-        <Block is="p" mb="16px">
-          Click this exercise link will bring you directly to an online IDE called codesandbox.io.
-        </Block>
-        <LessonButton defaultTab={defaultTab} path={path} />
-      </ContentSection>
+      {secondaryExerciseUrl ? (
```

Comment:
Looks like we're using the same `ContentSection` and props regardless of the value of `secondaryExerciseUrl`. Could we only conditionally change the children of `ContentSection` instead of changing `ContentSection`?

Reply:
Good catch!